osi_environment.proto

```diff
@@ -110,7 +110,16 @@ message EnvironmentalConditions
     }
 
     // Definition of ambient illumination states.
+    // Ambient light is any light in the vehicle's environment not emitted by the vehicle 
+    // itself. It can include sun/moon light, light from other cars, street lights, etc.
+    // 
+    // Lux is the SI unit of luminance or illumination of a one metre square area, which is 
+    // equal to one lumen per square meter[5]. 1lx = 1lm/m²
+    // The lumen (symbol: lm) is the SI derived unit of luminous flux, a measure of the total
+    // quantity of visible light emitted by a source. The lumen is defined in relation to the candela as 1lm=1cd.sr
     //
+    // \par Reference:
+    // [5] [The NIST Reference on Constants, Units, and Uncertainty](https://physics.nist.gov/cuu/Units/units.html)
```

Comment:
The references for the "how/why and from where did I pick the limits for the different levels are missing. The PDF in the issue is readable in Github, but can not be accessed by a user of OSI.

Reply:
This is just an initial classification. If it is acceptable I can add the references. These would be the European, and North American standars as the classification is based mostly on lighting standards for roads.